packages/steps/package.json

```
@@ -48,6 +48,7 @@
     "node": ">=16.0.0"
   },
   "dependencies": {
+    "@expo/eas-build-job": "^1.0.71",
```

I think that we should do the exact version as for other places
```suggestion
    "@expo/eas-build-job": "1.0.71",
```
so `lerna` can auto bump it when publishing a new release of `@expo/eas-build-job`.

BTW I don't see any changes to the `steps` package except this `package.json` change. Why is it needed?

---

👍 I didn't realize I committed that.

This is necessary so that users don't need to include both steps and eas-build-job in their custom functions to get staticContext.job typing, which only exists in eas-build-job.